### ISSUE TITLE
cannot resolve the dependency 'paper' ?

### ISSUE THREAD

**Issue by amgoing** [yemount/pose-animator#12]
an error("cannot resolve the dependency 'paper' ") occured when I try to use the command "yarn watch". I make sure that I use the 'npm install paper' command to install it. Maybe it was a redundant step, but I have no choice to do this. Could someone help me with this?
![error](https://user-images.githubusercontent.com/39852388/81543172-4b93fd00-93a8-11ea-9d8c-30cf2909c2fa.png)

**Comment by LinkXSystem** [comment]
I also encountered the same problem, did you solve it? @amgoing

**Comment by keshamin** [comment]
Same thing.

**Comment by serapth** [comment]
Open package.json
Change the paper line to:
"paper": "^0.12.1",

Then (for a later bug you will hit) while in package.json, go to the watch line and change it to:
"watch": "cross-env NODE_ENV=development parcel index.html --no-hmr --open --no-source-maps ",

The --no-source-maps is the key addition.  No run
yarn
then run 
yarn watch

You should be good now.

**Comment by amgoing** [comment]
wow~thank you so much！


—




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Mike"<notifications@github.com&gt;; 
发送时间: 2020年5月11日(星期一) 晚上10:34
收件人: "yemount/pose-animator"<pose-animator@noreply.github.com&gt;; 
抄送: "&amp;#x270E;&amp;#xFE4F;&amp;#xE53;.南枝"<1433397636@qq.com&gt;; "Mention"<mention@noreply.github.com&gt;; 
主题: Re: [yemount/pose-animator] cannot resolve the dependency &#39;paper&#39; ? (#12)





 
Open package.json
 Change the paper line to:
 "paper": "^0.12.1",
 
Then (for a later bug you will hit) while in package.json, go to the watch line and change it to:
 "watch": "cross-env NODE_ENV=development parcel index.html --no-hmr --open --no-source-maps ",
 
The --no-source-maps is the key addition.  No run
 yarn
 then run
 yarn watch
 
You should be good now.

**Comment by thllwg** [comment]
I made a PR (#15) that allows to run the application in a Docker container and fixes this problem.
You can build the container by running:
`docker build --pull --rm -f "Dockerfile" -t poseanimator:latest "."`
`docker run --rm -it  -p 1234:1234/tcp --device=/dev/video0:/dev/video0 poseanimator:latest`

**Comment by hithismani** [comment]
Optionally, if you're on windows and are facing an fsevents issue, just remove the yarn.lock and the node_modules file before trying @serapth's fixes. It should work perfectly :)

**Comment by amgoing** [comment]
yes! I did it! thanks!  :)------------------ 原始邮件 ------------------
发件人: "Mani Kumar"<notifications@github.com>
发送时间: 2020年5月13日(星期三) 凌晨2:20
收件人: "yemount/pose-animator"<pose-animator@noreply.github.com>;
抄送: "AmGoing"<1433397636@qq.com>;"Mention"<mention@noreply.github.com>;
主题: Re: [yemount/pose-animator] cannot resolve the dependency 'paper' ?(#12)




 
Optionally, if you're on windows and are facing an fsevents issue, just remove the yarn.lock and the node_modules file before trying @serapth's fixes. It should work perfectly :)
 
&mdash;

**Comment by hvitis** [comment]
Works great with your solution. Thank you @serapth

**Comment by Sba-Stuff** [comment]
If any of you got still issues, please go to third_party/paper/package.json and use version mentioned in that "package.json" file. Rest i followed what serapth said. Using Hardcode versions may cause issues.


![image](https://user-images.githubusercontent.com/36412543/84570347-6f32d480-ada6-11ea-98bf-28454e0d317e.png)

![image](https://user-images.githubusercontent.com/36412543/84570402-bf119b80-ada6-11ea-8307-b4d062af135f.png)

**Comment by isharaNilnuwan** [comment]
thanks, @serapth for the solution
